src/onegov/org/views/newsletter.py

```diff
@@ -99,6 +112,51 @@ def news_by_newsletter(
     return request.exclude_invisible(query.all())
```

Comment:
This will still exclude invisible content for the cronjob, so you should skip this step now.

But you should also take publication into account. Right now even news that are set to be published in the future or have ended publication would be included. You should be able to filter it within the query via `News.published`.

Reply:
I thought I covered it with a manual test, but no. The added test reproduces what you say